src/main/java/uk/ac/cam/cl/dtg/segue/etl/SchoolIndexer.java

```diff
@@ -124,6 +124,7 @@ synchronized void indexSchoolsWithSearchProvider() throws UnableToIndexSchoolsEx
                     School schoolToSave = new School(schoolArray[fieldNameMapping.get(Constants.SCHOOL_URN_FIELDNAME)],
                             schoolArray[fieldNameMapping.get(Constants.SCHOOL_ESTABLISHMENT_NAME_FIELDNAME)],
                             schoolArray[fieldNameMapping.get(Constants.SCHOOL_POSTCODE_FIELDNAME)],
+                            "t".equals(schoolArray[fieldNameMapping.get(Constants.SCHOOL_CLOSED_FIELDNAME)]),
```

Comment:
?!

Reply:
I have a column with `t`s and `f`s in, and I need boolean values. I will add a comment!